### DISCUSSION TITLE
Welcome to hugo-creative-portfolio-theme Discussions!

### DISCUSSION THREAD

**Discussion by kishaningithub** [kishaningithub/hugo-creative-portfolio-theme#98] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by katyakhvatkova** [comment]
Hello! I wanted to ask how to add additional sidebar menus to this website. I will describe what I would like to do: 

In the ```config.yaml``` file, under ```navlinks:``` I have added the following code: 
  ```navlinks:```
    ```- home: yes```
      ```name: Home```
      ```url: about/```
    ```- name: Education```
      ```url: education/```
    ```- name: Research```
      ```url: portfolio/```
    ```- name: Teaching```
      ```url: teaching/   ```
    ```- name: Contact```
      ```url: contact/```

And I have created directories inside the content folder called ```education/``` and and ```teaching/```. Inside these directories, I have duplicated the ```_index.md``` file inside the ```about/``` folder. My goal is to have these new folders have the same aesthetic layout as the ```_index.md``` file inside the ```about/```, only changing the written text. 

Right now, the folders appear in the side menu, but when I click on them the display is blank white. What other files do I need to change in order to add folders, and what lines of code do I need to add or change? (I don't have a lot of HTML experience, and I am seeing that there is a ```sidebar.html``` file inside that path ```/themes/hugo-creative-portfolio-theme/layouts/partials```. 

Please let me know if you need more descriptions of what I'm trying to troubleshoot!

**Comment by minhtriethong** [comment]
I would like to add a profile picture to sidebar, are there anyway to implement that ?

Thank you
